testing/tests/DevExpress.ui.widgets.form/form.API.option_formData.tests.js

```diff
@@ -329,3 +329,118 @@ QUnit.test('Reset editor\'s value when set formData: {dataField1: a}', function(
         assert.deepEqual(editor.option('value'), editor._getDefaultOptions().value, `a default value of the ${dataField} editor`);
     });
 });
+
+
+QUnit.module('Checkbox editor field', () => {
+    function checkCheckboxAndFormValue(form, dateField, editorValue, formValue) {
+        const editor = form.getEditor(dateField);
+        QUnit.assert.strictEqual(editor.option('value'), editorValue, `editor has ${editorValue} value`);
+
+        const expectedFormValue = formValue === 'no member' ? undefined : formValue;
+        QUnit.assert.strictEqual(form.option(`formData.${dateField}`), expectedFormValue, `formData has ${expectedFormValue} value`);
+
+        const $checkBox = $(editor.element());
```

Comment:
I would prefer to avoid these checks: we already checked editor.option('value') and it means that visual styles/css classes are applied correctly.

Reply:
Yep, removed this checks